sdk/communication/azure-communication-chat/README.md

```diff
@@ -478,12 +478,12 @@ if retry:
 ### Remove thread participant
 
 Use `remove_participant` method to remove thread participant from the thread identified by threadId.
-`user` is the `CommunicationUserIdentifier` you created by CommunicationIdentityClient.create_user() from User Access Tokens
+`identifier` is the `CommunicationUserIdentifier` you created by CommunicationIdentityClient.create_user() from User Access Tokens
 <!-- [User Access Tokens](#user-access-tokens)  -->
 and was added into this chat thread.
-- Use `user` to specify the `CommunicationUserIdentifier` you created
+- Use `identifier` to specify the `CommunicationUserIdentifier` you created
 ```python
-chat_thread_client.remove_participant(user=new_user)
+chat_thread_client.remove_participant(identifier=new_user)
 
 # # converesely you can also do the following; provided the user_id is known
 # from azure.communication.identity import CommunicationUserIdentifier
```

Comment:
Python uses dashes
```suggestion
# from azure-communication-identity import CommunicationUserIdentifier
```
Actually, you wouldn't import from identity because that type is included in Chat right?

On line 492 CommunicationUserIdentifier is missspelled

Reply:
Yep, you're correct. All the identifiers are now being exposed from `chat`. I have modified the import statement